src/Makefile.am

```diff
@@ -75,11 +75,21 @@ libfapolicyd_la_SOURCES += \
 endif
 
 if WITH_DEB
-libfapolicyd_la_SOURCES += library/deb-backend.c
+libfapolicyd_la_SOURCES += library/deb-backend.c \
+	library/md5-backend.c \
+	library/md5-backend.h
 fapolicyd_CFLAGS += -DLIBDPKG_VOLATILE_API
 fapolicyd_LDFLAGS += -ldpkg
 endif
 
+if WITH_EBUILD
+libfapolicyd_la_SOURCES += library/ebuild-backend.c \
+	library/md5-backend.c \
```

Comment:
You're including the md5-* stuff twice if building with both Debian and ebuild support, I think?

Reply:
Yes, we probably need a meta WITH_MD5 that's set if deb or ebuild.

Alternately a build system designed sometime in the last quarter of a century.